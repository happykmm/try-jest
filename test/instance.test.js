const foo = jest.fn();

foo();
new foo();
new foo();

test('instance', () => {
  expect(foo).toMatchSnapshot();
  expect(foo.mock.instances).toMatchSnapshot();
  console.log(foo.mock.instances.length);
});

