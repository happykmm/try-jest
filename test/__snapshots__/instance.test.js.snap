// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`instance 1`] = `
[MockFunction] {
  "calls": Array [
    Array [],
    Array [],
    Array [],
  ],
}
`;

exports[`instance 2`] = `
Array [
  undefined,
  mockConstructor {},
  mockConstructor {},
]
`;
